doc/classes/Object.xml

```diff
@@ -601,43 +731,45 @@
+				The [param n] is the number, or amount, of the message's subject. It is used by the translation system to fetch the correct plural form for the current language.
+				For detailed examples, see [url=$DOCS_URL/tutorials/i18n/localization_using_gettext.html]Localization using gettext[/url].
+				[b]Note:[/b] Negative and [float] numbers may not properly apply to some countable subjects. It's recommended handling these cases with [method tr].
 			</description>
 		</method>
 	</methods>
 	<signals>
 		<signal name="property_list_changed">
 			<description>
+				Emitted when [method notify_property_list_changed] is called.
 			</description>
 		</signal>
 		<signal name="script_changed">
 			<description>
-				Emitted whenever the object's script is changed.
+				Emitted when the object's script is changed.
+				[b]Note:[/b] When this signal is emitted, the new script is not initialized yet. If you need to access the new script, defer connections to this signal with [constant CONNECT_DEFERRED].
 			</description>
 		</signal>
 	</signals>
 	<constants>
 		<constant name="NOTIFICATION_POSTINITIALIZE" value="0">
-			Called right when the object is initialized. Not available in script.
+			Notification received when the object is initialized, before its script's [method _init] is called. Used internally.
 		</constant>
 		<constant name="NOTIFICATION_PREDELETE" value="1">
-			Called before the object is about to be deleted.
+			Notification received when the object is about to be deleted. Can act as the deconstructor of some programming languages.
 		</constant>
 		<constant name="CONNECT_DEFERRED" value="1" enum="ConnectFlags">
-			Connects a signal in deferred mode. This way, signal emissions are stored in a queue, then set on idle time.
+			Deferred connections trigger their [Callable]s on idle time, rather than instantly.
 		</constant>
 		<constant name="CONNECT_PERSIST" value="2" enum="ConnectFlags">
-			Persisting connections are saved when the object is serialized to file.
+			Persisting connections are stored when the object is serialised (such as when using [method PackedScene.pack]). In the editor, connections created through the Node dock are always persisting.
 		</constant>
 		<constant name="CONNECT_ONE_SHOT" value="4" enum="ConnectFlags">
 			One-shot connections disconnect themselves after emission.
 		</constant>
 		<constant name="CONNECT_REFERENCE_COUNTED" value="8" enum="ConnectFlags">
-			Connect a signal as reference-counted. This means that a given signal can be connected several times to the same target, and will only be fully disconnected once no references are left.
+			Reference-counted connections can be assigned to the same [Callable] multiple times. Calling the [method disconnect] method subtracts one connection at the time, until no connections remain.
```

Comment:
English is not my native language. I think "until no connections remain" is redundant and sounds wrong to me.

Reply:
To me, `"will only be fully disconnected once no references are left"` makes sense because it means something like this:

```
if current_reference_count <= 0:
    you_are_fully_disconnected()
```

Although kind of silly, I think `"Calling the method subtracts one connection at the time, until no connections remain"` can be interpreted as:

```
func method():
    while still_have_connections():
        connection -= 1
```

Reply:
I agree with @timothyqiu . It may sound like "babyfying" the explanation, but CONNECT_REFERENCE_COUNTED is a pretty simple concept with a deserved simple explanation, that can be easily tested to be fully understood.

Reply:
Not sure about _preferring_ the previous explanation, but I agree on the principle.

Reply:
Accidentally pushed early, but the description of this one has been changed. You may review again.